llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -1799,8 +1799,7 @@ void MemoryDepChecker::mergeInStatus(VectorizationSafetyStatus S) {
 ///     }
 static bool isSafeDependenceDistance(const DataLayout &DL, ScalarEvolution &SE,
                                      const SCEV &MaxBTC, const SCEV &Dist,
-                                     uint64_t MaxStride,
```

Comment:
I think the comment above needs updating now that you've removed `TypeByteSize` and `MaxStride` is now in bytes.